pkg/services/gitopswriter/gitopswriter.go

```diff
@@ -102,7 +102,7 @@ func (dw *gitOpsDirectoryWriterSvc) AddApplication(ctx context.Context, app mode
 		Description:   fmt.Sprintf("Added yamls for %s", app.Name),
 		CommitMessage: AddCommitMessage,
 		TargetBranch:  defaultBranch,
-		NewBranch:     automation.GetAppHash(app),
+		NewBranch:     automation.GetRandomName("wego"),
```

Comment:
Why wasn't AppHash enough here? Other than the annoying thing that you needed to remove the branch constantly. :D I guess that's why?

Reply:
If you use the app hash, you can only have one PR for an app at a time...which I guess is fine for some cases...

Reply:
`GetAppHash`  doesn't generate a string that is actually random + unique! It converts the input using `md5.Sum()`, hence the duplicates: https://github.com/weaveworks/weave-gitops/blob/main/pkg/services/automation/generator.go#L393-L395

Reply:
can we not use UUID then instead? 
`NewBranch:     uuid.New().String(),` 🤔

Reply:
@bigkevmcd?

Reply:
I don't feel too strongly about this, if UUIDs are better for the user-experience (which is what I think @Himangini  is implying) I'm fine with that too.

Reply:
Let's use `uuid.New().String()` since it's the easiest thing and wait for product/UX feedback. :)

Reply:
Along with the UX improvement, I was thinking this saves us from writing our own string generator and then testing it + less code to maintain 😅